browser/app/less-2/inc/alert.less

```diff
@@ -0,0 +1,68 @@
+.alert {
+    border: 0;
+    position: fixed;
+    max-width: 500px;
+    margin: 0;
+    box-shadow: 0 4px 5px rgba(0, 0, 0, 0.1);
+    color: @white;
+    width: 100%;
+    right: 20px;
+    border-radius: 3px;
+    padding: 17px 50px 17px 17px;
+    z-index: 10010;
+    .animation-duration(800ms);
+    .animation-fill-mode(both);
+
+    &:not(.progress) {
+        top: 20px;
+
+        @media(min-width: (@screen-sm-min)) {
+            left: 50%;
+            margin-left: -250px;
+        }
+    }
+
+    &.progress {
+        bottom: 20px;
+        right: 20px;
+    }
+
+    &.alert-danger {
+        background: @red;
+    }
+
+    &.alert-success {
+        background: @green;
+    }
+
+    &.alert-info {
+        background: @blue;
+    }
+
+    @media(max-width: (@screen-xs-max)) {
+        left: 20px;
+        width: ~"calc(100% - 40px)";
+        max-width: 100%;
+    }
+
+    .progress {
+        margin: 10px 10px 8px 0;
+        height: 5px;
+        box-shadow: none;
+        border-radius: 1px;
+        background-color: @blue;
+        border-radius: 2px;
+        overflow: hidden;
+    }
+
+    .progress-bar {
+        box-shadow: none;
+        background-color: @white;
+        height: 100%;
+    }
+
+    .close {
```

Comment:
![close_icon](https://user-images.githubusercontent.com/13393018/28408591-9db7f824-6d55-11e7-87e6-0e8215f8e3b4.png)

![close_icon_2](https://user-images.githubusercontent.com/13393018/28408710-000a75a6-6d56-11e7-9a64-7af5ba212769.png)


This is how it's rendered in my browser (MacOS Chrome, Firefox and Safari). Are you on a Windows machine?